source/code/include/scxcorelib/scxatomic.h

```diff
@@ -113,6 +115,22 @@ __inline static bool scx_atomic_decrement_test(scx_atomic_t* v)
 #endif
 
 #elif defined(linux)
+
+// Use intrinsics for IBM powerPC
+#if defined(ppc)
+
+static __inline__ void scx_atomic_increment(scx_atomic_t* v)
+{
+    ++(*v);
```

Comment:
This is using [c++ 11 atomics ](http://en.cppreference.com/w/cpp/atomic/atomic/operator_arith)
We do get a warning on osdevppc-rhel71-05 that the test should be run on a multiple CPU machine although `lscpu` shows 32 CPUs. Will investigate some more.